content/en/getting_started/integrations/prometheus.md

```diff
@@ -22,25 +22,29 @@ aliases:
   - /getting_started/prometheus
 ---
 
-Collect your exposed Prometheus metrics from your application [running inside containers](#container-auto-discovery) or directly [on your host](#host-agent) using the Datadog Agent and the [Datadog-Prometheus][1] integration.
+Collect your exposed Prometheus and OpenMetrics metrics from your application [running inside containers](#container-auto-discovery) or directly [on your host](#host-agent) by using the Datadog Agent, and the [Datadog-OpenMetrics][1] or [Datadog-Prometheus][2] integrations.
 
 ## Overview
 
-Starting with version 6.1.0, the Agent includes a [Prometheus][2] check capable of scraping Prometheus endpoints with a few lines of configuration. This page explains the basic usage of the generic `Prometheus` check, which enables you to scrape custom metrics from Prometheus endpoints. For more advanced usage of the `PrometheusCheck` interface, including writing a custom check, see the [Developer Tools][3] section.
+Starting with version 6.5.0, the Agent includes [OpenMetrics][3] and [Prometheus][4] checks capable of scraping Prometheus endpoints with a few lines of configuration. This page explains the basic usage of these checks, which enable you to scrape custom metrics from Prometheus endpoints. 
+
+We recommend using the OpenMetrics check since it is more efficient and fully supports Prometheus text format. Use the Prometheus check only when the metrics end-point does not support a text-format.
```

Comment:
```suggestion
Datadog recommends using the OpenMetrics check since it is more efficient and fully supports Prometheus text format. Use the Prometheus check only when the metrics endpoint does not support a text format.
```